modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/import/ImportSubmit.js

```diff
@@ -21,18 +21,26 @@ import React, {useCallback, useState} from 'react';
 import ImportModal from './ImportModal';
 
 function ImportSubmit({
+	disabled,
 	evaluateForm,
+	fieldsSelections,
+	fileContent,
+	fileFields,
 	formDataQuerySelector,
 	formImportURL,
 	formIsValid,
 	formIsVisible,
 	portletNamespace,
+	setFileContent,
 }) {
-	const [visible, setVisible] = useState(false);
+	const [visibleModalPreview, setVisibleModalPreview] = useState(undefined);
+	const [startImport, setStartImport] = useState(undefined);
```

Comment:
`undefined` represents a "non-defined" value: assigning it a variable doesn't represent a common scenario. I think `false` will fit better in this one.